packages/tools-iottwinmaker/src/commands/deploy.ts

```diff
@@ -84,11 +110,24 @@ export const handler = async (argv: Arguments<Options>) => {
         const filtered_property_definitions = Object.entries(propertyDefinitions).reduce((acc, [key, value]) => {
           if (!value['isInherited']) {
             acc[key] = value;
+          } else if ('defaultValue' in value) {
+            acc[key] = { defaultValue: value['defaultValue'] };
           }
           return acc;
         }, {} as { [key: string]: object });
         componentTypeDefinition['propertyDefinitions'] = filtered_property_definitions;
       }
+      // remove inherited functions
+      const componentTypeFunctions = componentTypeDefinition['functions'] as object;
```

Comment:
`as object` is almost the same as saying `as any`. there must be something we know more about it than it being any object?

Reply:
fixed same as above, thanks!